crates/forge/tests/it/fuzz.rs

```diff
@@ -203,10 +203,10 @@ contract FuzzerDictTest is Test {
     .unwrap();
 
     // Test that immutable address is used as fuzzed input, causing test to fail.
-    cmd.args(["test", "--fuzz-seed", "100", "--mt", "testImmutableOwner"]).assert_failure();
+    cmd.args(["test", "--fuzz-seed", "119", "--mt", "testImmutableOwner"]).assert_failure();
```

Comment:
Any reason for this specific change?

Reply:
looks like contract bytecode has been changed and it no longer fails in configured runs, changing the seed keeps the runs to catch it but can also bump the runs with prev seed